media-gfx/blender/blender-2.83.15.ebuild

```diff
@@ -123,6 +130,13 @@ blender_check_requirements() {
 	fi
 }
 
+blender_get_version() {
+	# Get blender version from blender itself.
+	BV=$(grep "BLENDER_VERSION " ${S}/source/blender/blenkernel/BKE_blender_version.h | cut -d " " -f 3)
```

Comment:
```suggestion
	BV=$(grep "BLENDER_VERSION " ${S}/source/blender/blenkernel/BKE_blender_version.h | cut -d " " -f 3 || die)
```

Reply:
Let's check for potential grep failure too
```suggestion
	BV=$(grep "BLENDER_VERSION " ${S}/source/blender/blenkernel/BKE_blender_version.h | cut -d " " -f 3; assert)
```

Reply:
`${S}` is needed when this is called from src_unpack to get the correct svn checkout url.
I'll add a TODO so we don't forget to remove it after we host a tar ball for the non live versions.

Reply:
Actually I'll just use SLOT instead so I don't need to call this in the unpack stage.